lib/open_project/pdf_export/engine.rb

```diff
@@ -38,7 +38,7 @@ class Engine < ::Rails::Engine
       menu :admin_menu,
             :export_card_configurations,
             {:controller => '/export_card_configurations', :action => 'index'},
-            {:caption    => :'label_export_card_configuration_plural', :html => {:class => "icon2 icon-tracker"}}
+            {:caption    => :'label_export_card_configuration_plural', :html => {:class => "icon2 icon-ticket-down"}}
```

Comment:
Line is too long. [117/80]<br>Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.